test/controllers/transportEquipment/index/GoodsReferenceControllerSpec.scala

```diff
@@ -22,23 +22,23 @@ import forms.SelectableFormProvider
 import generators.Generators
 import models.reference.Item
 import models.{Index, NormalMode, SelectableList}
-import pages.transportEquipment.index.GoodsReferencePage
+import pages.transportEquipment.index.ItemPage
 import play.api.test.FakeRequest
 import play.api.test.Helpers._
-import viewModels.transportEquipment.index.GoodsReferenceViewModel
+import viewModels.transportEquipment.SelectItemsViewModel
 import views.html.transportEquipment.index.GoodsReferenceView
 
 class GoodsReferenceControllerSpec extends SpecBase with AppWithDefaultMockFixtures with Generators {
 
-  private val viewModel: GoodsReferenceViewModel = GoodsReferenceViewModel(emptyUserAnswers, None)
+  private val viewModel: SelectItemsViewModel = SelectItemsViewModel(emptyUserAnswers, None)
 
   private val items = SelectableList(Seq(Item(123, "description")))
 
   private val formProvider = new SelectableFormProvider()
   private val mode         = NormalMode
   private val form         = formProvider(mode, "transport.equipment.selectItems", items)
 
-  private lazy val controllerRoute = controllers.transportEquipment.index.routes.GoodsReferenceController.onPageLoad(arrivalId, Index(0), mode).url
+  private lazy val controllerRoute = controllers.transportEquipment.index.routes.GoodsReferenceController.onPageLoad(arrivalId, Index(0), itemIndex, mode).url
```

Comment:
should this not be goodsReferenceIndex? I l know it's Just Index(0) from SpecBase but may be worth changing and adding val  goodsReferenceIndex = Index(0) to SpecBase then use it in the controller specs

Reply:
Yeah we can change everything to use ItemIndex now, there is no distinction between goodsReferenceIndex and ItemIndex anymore